main/src/library/Int32.flix

```diff
@@ -0,0 +1,88 @@
+namespace Int32 {
+
```

Comment:
Are we implicitly assuming that Int32 represents signed two's-complement integers?

Or do we sometimes want unsigned integers as well?